Main

elingv opened the pull request.

elingv committed:
created index.html

elingv committed:
created htmls pages

elingv committed:
chang file name

elingv committed:
rename dogs.html

elingv committed:
change bkg color

elingv committed:
change to one blog class, same style for all

elingv committed:
style css

elingv committed:
add css

elingv committed:
fix image css path

elingv committed:
Update master.css

elingv committed:
Update master.css

elingv committed:
move to main branch

elingv committed:
change nav color #0f52ba

elingv committed:
fix meta tag

elingv committed:
update css

elingv committed:
wired links on index page

elingv committed:
try to add section tag

tylervanbrocklin commented on blog.html:
```diff
@@ -0,0 +1,77 @@
+
+    <main>
+      <h2>Adoptadog Blog</h2>
+        <section class="blog">
```
In this case you'll want to use div tags to wrap each blog post. We typically only use section for specific sections or areas of your site. You might use a section to wrap all of your blog posts, but each individual post should be wrapped and styled in a div tag.

tylervanbrocklin commented on blog.html:
```diff
@@ -0,0 +1,77 @@
+      <h2>Adoptadog Blog</h2>
+        <section class="blog">
+          <img src="images/blog-1.jpg" width="300" alt="a woman holding a dog facing the grand canyon">
+          <h2>Traveling With Your Dog</h2>
```
Would suggest wrapping your h2 and p tags in another div. This will allow you to push the text onto the side of your images

tylervanbrocklin commented on dogs.html:
```diff
@@ -0,0 +1,174 @@
+      <img src="images/cart-icon.svg" width="24" alt="shopping cart icon">
+      <p>$1234.56</p>
+      <button type="button">Checkout</button>
+
```
Make sure to copy your header from index.html onto your other pages

tylervanbrocklin commented on dogs.html:
```diff
@@ -0,0 +1,174 @@
+
+    <main>
+      <h2>Here are all of our Dogs!</h2>
+        <section>
```
You did a great job structuring this page. You'll want to create a class, maybe called dog-cards and use it to create your grid. You can use flexbox to create the grid, or css grid. Checkout this reference on using flexbox: https://css-tricks.com/dont-overthink-flexbox-grids/

This will allow you to create the grid of tiles seen in the PDF example we provide in your project older

elingv committed:
format css

elingv committed:
format header/footer for all pages

elingv committed:
format blog css

elingv committed:
in process of style dogs.html page

elingv committed:
update css

elingv committed:
fix header with flexbox

elingv committed:
update header flexbox

elingv committed:
add js to the site

elingv committed:
add js to dogcard

elingv committed:
card js array in progress

elingv committed:
editing js dogcard

elingv committed:
fix blog h2 margin top

elingv committed:
onClick js

elingv committed:
fix blog css spacing

elingv committed:
js for adopt button

Commit:
add css for checkout page

Commit:
style form css

Commit:
format checkout form and header checkout btn

Commit:
style checkout form

Commit:
dog card adopt cost js

Commit:
responsive assignment

Commit:
modify dogs card css

Commit:
remove div nav class css

Commit:
flexbox dog card

Commit:
assignment 6 index page layout

Commit:
media query css

Commit:
media query css

Commit:
responsive css

Commit:
checkout form css

Commit:
adjust header flexbox right

Commit:
add gradient header

Commit:
css fix

jimmynono commented on index.html:
```diff
@@ -0,0 +1,180 @@
+            </div>
+          </div>
+
+        <div ID="content-right">
```
Don't use id's for this. Use classes. Always classes for CSS.

Commit:
checkout form css issue

Commit:
fix css

Commit:
@media css

Commit:
fix media query reponsive layout

Commit:
dog card mobile responsive

Commit:
align to master.css

Commit:
css

Commit:
mobile css

Commit:
media query css

Commit:
checkout form css

Commit:
add card-sm class

Commit:
edit button link

Commit:
fix form css

Commit:
checkout page css

Commit:
form css fix

Commit:
add blog @media css

Commit:
for js DOM assignment

Commit:
blog js dom

Commit:
delete 3.js

Commit:
forloop console log

Commit:
consolelog form value

Commit:
remove test pages

Commit:
reinstate form-dynamic.js file

elingv committed:
Update checkout.html

Trying to fix - Loading failed for the <script> with source “https://elingvoon.github.io/html200-adopt-a-dog/scripts/form-dynamic.js”.

jimmynono commented on checkout.html:
```diff
@@ -0,0 +1,141 @@
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="./styles/master.css">
+    <script src="./scripts/form-dynamic.js" type="text/javascript">
```
For this to work you'll need to move this to the bottom of your body

elingv committed:
Rename dogCard.js to scripts/dogCard.js

Commit:
moved dogs.js to scripts folder

Commit:
moved dogs.js to scripts folder

elingv committed:
Update index.html

Commit:
unmerge edits

Commit:
shopping cart icon sm

Commit:
checkout total implemented

Commit:
jquery dogcard effect

Commit:
form console log

Commit:
remove labels as they are unnecessary

Commit:
refactor form submit w/ jquery

Commit:
fix hover and target cardtotal class

Commit:
fix css

Commit:
add readme and testing files

elingv committed:
Update README.md

elingv committed:
Update testing.txt

Commit:
add btn css, active,hover

Commit:
merge with remote

Commit:
adjust checkout form width for mobile

Commit:
mobile checkout width tweak

Commit:
console log form

Commit:
form css edit

elingv commented:
Yay!!!
